docs/getting-started.txt

```diff
@@ -1,300 +1,311 @@
-is ``1`` by default; setting it to a larger value enables distributed training. If
-``slots_per_trial`` is larger than the number of slots on any single agent machine, you should set
-``resources.slots_per_trial`` to a multiple of the number of GPUs in each machine in the cluster.
-For example, if your cluster has 8-GPU machines, you should use a value such as 8, 16, 24, etc. If
-using a Determined cluster deployed in the cloud, by default each agent will have eight GPUs.
+   In this example, you can see two experiments. Experiment 3 is in the COMPLETED state and
+   experiment 4 is in the ACTIVE state.
+
+   While an experiment is in the ACTIVE, training state, click an experiment tile to see the Metrics
+   graph update with your currently defined metrics. In this example, the graph displays the loss
+   and error rate per batch:
+
+   .. image:: /assets/images/qs04.png
+      :width: 704px
+      :align: center
+      :alt: Metrics graph detail
 
-Below is the ``distributed.yaml`` file, which is very similar to the ``const.yaml`` file except we
-now set ``resources.slots_per_trial``.
+#. When the experiment completes, click an experiment tile to view the trial page:
+
+   .. image:: /assets/images/qs03.png
+      :width: 704px
+      :align: center
+      :alt: Trial page
+
+   Because this experiment trains a single model with a fixed set of hyperparameters, there is only
+   one trial so selecting the experiment tile takes you directly to the trial page. [TBD: what does
+   this mean?]
+
+You can create additional experiments repeating the ``det experiment create`` command as a way to
+gain familiarity with other Determined WebUI functions.
+
+********************************
+ Run a Distributed Training Job
+********************************
+
+This exercise requires a Determined cluster with multiple GPUs.
+
+Determined can coordinate multiple GPUs to train a single trial more quickly by using distributed
+training. Distributed training is best applied to complex models. The model used in this example
+does not fully demonstrate the benefits of distributed training but is still useful in showing how
+to work with additional hardware resources.
+
+Moving from single-GPU training to distributed training involves changing a single
+:ref:`resources.slots_per_trial <exp-config-resources-slots-per-trial>` setting in the experiment
+``distributed.yaml`` configuration file, which is similar to the ``const.yaml`` file used in the
+previous example, except that a *resources* property is added with the ``resources.slots_per_trial``
+field is set to ``8`` to represent eight GPUs. The default ``slots_per_trial`` setting is ``1``. If
+``slots_per_trial`` is larger than the number of slots on an agent, set
+``resources.slots_per_trial`` to a multiple of the number of GPUs in each machine in the cluster.
+For example, if your cluster consists of 8-GPU machines, use a value that is a multiple of eight.
```

Comment:
suggestion: don't phrase as conditional action "If X is Y, then set X to Z".  Maybe express as a property, like "For clusters that have multiple GPUs per machine, slots_per_trial must be divisible by the number of GPUs per machine or it will fail to be scheduled.  For instance, a 7 GPU job can't be scheduled on a 4-GPU per machine cluster, but an 8 GPU job can."